### PR TITLE
New Rockplanet ruin, Budgetcuts!

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
This PR adds a Nanotrasen Research facility, in a bit of a pickle.

Loot:
Combat supplies ( 1 m1911 / energy crossbow / security locker / security vendor / hos locker )
Medical supplies ( compact defib / brute,burn,oxy kit ) 
Kitchen supplies
Research supplies ( sci protolathe / circuit imprinter / experimentor / slime cores / research note / nanite supplies )
Fake Death ripley
jackhammer

Threats:
Emergency Retirement Team ( Assault officer / 2x SMG officers / 2x PISTOL officers / 2x FIST officers )
Emergency Treasure Retrieval Team ( RANGED pirate / 2x SWORD pirates / HARDSUIT pirate ) 

Lore grimble!
As you know, Nanotrasen hasn't exactly had the most economically successful season. Due to this, theres a need for some.... Budget cuts _(get it? its the name! im really smart)_. This research station hasn't exactly been productive lately, nor has it been worth having a research station in a place as remote as this. Nanotrasen is now left with two choices. 
Pay the researchers their promised bonus, or... Well, you can guess the second option. 
The Research Director fearing what Nanotrasen would do to him sends out an SOS and kills himself.

Sadly for the rest of the crew, nobody responds to the SOS call, sealing their fate. A certain group of pirates, however, locate the signal and prepare to raid it. 

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
More ruins = good! more diversity in ruins is also good!

## Changelog

:cl:
add: Added a new Rock ruin, rockplanet_budgetcuts
/:cl:

### CODE REVIEW

File: code/datums/ruins/rockplanet.dm [hunk 1]
```diff
@@ -86,3 +86,9 @@
 	description = "A factory made for HARM and AGONY."
 	id = "rockplanet_harmfactory"
 	suffix = "rockplanet_harmfactory.dmm"
+
+/datum/map_template/ruin/rockplanet/budgetcuts
+	name = "Budgetcuts"
+	description = "Nanotrasen's gotta lay off some personell, and this facility hasn't been worth the effort so far"
```
Comment (Erikafox) [inline code/datums/ruins/rockplanet.dm]: personnel

File: code/game/area/areas/ruins/rockplanet.dm [hunk 2]
```diff
@@ -6,3 +6,11 @@
 /area/mine/rockplanet_nanotrasen
 	name = "Abandoned Mining Facility"
 
+//rockplanet_budgetcuts
+
+/area/mine/rockplanet
```
Comment (Erikafox) [inline code/game/area/areas/ruins/rockplanet.dm]: bad ruin area path, check the other ruin area .dms and repath accordingly.

File: _maps/map_catalogue.txt [hunk 3]
```diff
@@ -230,6 +230,10 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 
 
 	RockRuins:
+		File Name = "_maps\RandomRuins\RockRuins\rockplanet_budgetcuts.dmm"
+		Size = (x = 52)(y = 44)(z = 1)
+		Tags = "hard combat Challenge", "Medium Loot", "hospitable"
```
Comment (Erikafox) [inline _maps/map_catalogue.txt]: capitalize

File: _maps/RandomRuins/RockRuins/rockplanet_budgetcuts.dmm [hunk 4]
```diff
@@ -0,0 +1,5319 @@
...
+/obj/effect/decal/cleanable/garbage,
+/obj/effect/decal/cleanable/generic,
+/turf/open/floor/plasteel,
+/area/mine/rockplanet)
+"aI" = (
+/obj/structure/filingcabinet/chestdrawer,
+/obj/item/paper/crumpled/awaymissions/research,
+/obj/item/paper/fluff/awaymissions/moonoutpost19/research/evacuation,
+/obj/item/paper/crumpled/awaymissions/research/sensitive_info,
+/obj/item/paper/fluff/awaymissions/moonoutpost19/research/xeno_queen,
```
Comment (Erikafox) [inline _maps/RandomRuins/RockRuins/rockplanet_budgetcuts.dmm]: read these papers and make sure they don't go into bizarro world, they're old.

**Review by Erikafox — COMMENTED** [review]
map review later

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
I have the odd feeling that one of these guys will bump the door, they'll see each other, and they'll fight.
StrongDMM_P5RZzlAnkq
ditch the consuming and industrial extracts. Greys for xenobio is good loot already.
StrongDMM_UhdhGNJq5O
swap for a helm console, just have it be broken.
StrongDMM_ceWQgYIS0q
Swap the explosive mine for a (sophisticated) shrapnel mine. Explosives are not really fun to deal with. Shrapnel has a bit more grace.
I also wouldn't be upset if you added more.
StrongDMM_dItupkaFek
what's up with this?
StrongDMM_1kAI6Y4Q8N
Decal

**Comment by Erikafox** [comment]
The pirates outside should be the suited variants, they'll die otherwise.

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Erikafox** [comment]
firefox_YSfXdEnTJL
The real authentic HoS locker needs to go, it has a secfab and enough gear for a small ship.

**Comment by Erikafox** [comment]
Need to fix the pirates on the outside.

**Comment by Erikafox** [comment]
fix the pirates

**Comment by Erikafox** [comment]
re-open when you fix the pirates.

**Comment by spockye** [comment]
GWA

**Review by Erikafox — APPROVED** [review]

**Pull request merged**